repositories/user_character.go

```diff
@@ -42,3 +44,30 @@ func (r *UserCharacterRepository) GetList(userId string) ([]models.UserCharacter
 
 	return userCharacters, nil
 }
+
+// ガチャ結果をusers_charactersテーブルにバルクインサートする
+func (r *UserCharacterRepository) InsertBulk(userId string, characters []models.UserCharacterInsert) error {
```

Comment:
UserCharacterTableのID生成はRepository（ここ）で行うことで、引数には `characterIDs []string` を受け取ってシンプルな設計にしましょう！

```suggestion
func (r *UserCharacterRepository) InsertBulk(userId string, characterIDs []string) error {
```

Reply:
https://github.com/ShinnosukeSuzuki/techtrain-mission-ca-tech-dojo-golang/pull/8/commits/037057034130c56ff7e051acca0eddf0ba7515d9
上記コミットで対応しました。
characterIDsと新しい配列を用意せず、characterID情報を持っているGachaResultの配列を引数にすることで、UserCharacterInsertを使わないように修正しました。